spec/integration/search/booster_spec.rb

```diff
@@ -2,21 +2,22 @@
 
 RSpec.describe 'BoosterTest' do
   it "service_manual_formats_are_weighted_down" do
-    commit_document("mainstream_test",
+    commit_document("govuk_test",
       "title" => "Agile is good",
       "link" => "/agile-is-good",
       "format" => "service_manual_guide",
     )
 
-    commit_document("mainstream_test",
+    commit_document("govuk_test",
       "title" => "Being agile is good",
       "link" => "/being-agile-is-good",
       "format" => "service_manual_topic",
     )
 
-    commit_document("mainstream_test",
+    commit_document("govuk_test",
       "title" => "Can we be agile?",
       "link" => "/can-we-be-agile",
+      "format" => "transaction",
```

Comment:
Transactions are boosted, which means this test isn't just testing service manuals. Could this be an unboosted format like `edition`?